woodwork/tests/demo_tests/test_retail.py

```diff
@@ -23,11 +23,11 @@ def set_testing_headers():
 
 def test_load_retail_to_accessor_diff():
```

Comment:
Saw you changed the name of the tests for `read_csv` to not include "to_accessor", which I liked. Should we do that here too?

Reply:
Yes, will update

Reply:
✅  bcd833a